content/projects/gsoc/2024/project-ideas/automating-rpu-for-jenkinsci-organization.adoc

```diff
@@ -0,0 +1,67 @@
+---
+layout: gsocprojectidea
+title: "Manage jenkinsci GitHub permissions as code"
+goal: "Automating the management of GitHub permissions for the jenkinsci organization"
+category: Tools
+year: 2024
+status: draft
+sig: infra
+skills:
+- Java
+- Groovy
+- git
+- Maven
+- SnakeYAML
+- Data extraction from GitHub repositories
+- GitHub user and team management
+mentors:
+- "notmyfault"
+links:
+  emailThread: https://community.jenkins.io/t/gsoc-2024-project-idea-manage-jenkinsci-github-permissions-as-code/11186
+  gitter: gsoc2024-rpu:matrix.org
+  meetings: /projects/gsoc/#office-hours
+---
+
+=== Background
+
+To manage artifactory permission, diverge between Jira and GitHub issues, and activate automatic releases, the jenkinsci organization uses a tool called https://github.com/jenkins-infra/repository-permissions-updater/[repository permission updater (RPU)].
+
+Despite the name containing "repository permission", the RPU can't update or manage repository permission at all.
```

Comment:
Managing permissions is most of what RPU does. The repository being referred to is the Maven repository at `repo.jenkins-ci.org` (note the subdomain).